decidim-core/spec/system/notifications_spec.rb

```diff
@@ -111,4 +111,45 @@
       end
     end
   end
+
+  context "with comment notifications" do
+    let!(:notification) { create :notification, :comment_notification, user: user }
+
+    before do
+      resource.destroy!
+      page.visit decidim.notifications_path
+    end
+
+    it "shows the comment notification with the conversation link" do
+      comment_id = notification.extra["comment_id"]
+      comment_definition_string = "commentId=#{comment_id}#comment_#{comment_id}"
+      links = page.all(".card.card--widget a")
+      hrefs = links.find { |link| link[:href].include?(comment_definition_string) }
+
+      expect(hrefs).not_to be(nil)
+    end
+
+    it "shows the comment in the notification" do
+      comment_notification = page.find(".card.card--widget")
+      comment_body = Decidim::Comments::Comment.first.body["en"]
+
+      expect(comment_notification).to have_content(comment_body)
+    end
+  end
+
+  context "with proposal notifications" do
```

Comment:
Can we move this spec to `decidim-proposals/spec/events/decidim/proposals/publish_proposal_event_spec.rb`?